Tests/demistomock/demistomock.py

```diff
@@ -1022,6 +1024,26 @@ def searchIndicators(fromDate='', query='', size=100, page=0, toDate='', value='
     Returns:
       dict: Object contains the search results
 
+    You can searchIndicators one batch at a time, without needing to load all indicators at once, by adding the argument
+    searchAfter after the demisto.executeCommand.
+    When you run a search for the query type:IP, the return value includes searchAfter
+    ```
+    >>> demisto.executeCommand(searchIndicators, "query": 'type:IP', "size" :1000})
+    {
+        "iocs": [],
+        "searchAfter": [1596106239679, dd7aa6abfcb3adf793922618005b2ad5],
+        "total": 7
+    }
+    ```
+    You can then use the returned value of searchAfter to iterate over all batches.
+    ```
+    >>> res = demisto.executeCommand("searchIndicators", {"query" : 'type:IP', "size" : 3})
```

Comment:
probably my bad, maybe we should set the size here to the default value (50) or another number we recommend (just not leave it as 3).